tools/Library.Settings.targets

```diff
@@ -32,7 +32,7 @@
     <OutputPath>bin\$(Configuration)\</OutputPath>
     <DocumentationFile>bin\$(Configuration)\$(AssemblyName).xml</DocumentationFile>
     <AdditionalBinariesFolder>$(LibraryFxTarget)\</AdditionalBinariesFolder>
-    <NoWarn>1591;1573;1570;1573</NoWarn>
+    <NoWarn>1591;1573</NoWarn>
```

Comment:
Heads up that this changes the ignored warnings for all libraries in the sdk repo.

IMO it's the right thing to do (if you have a 1570 like I did it will fail to put ANY of the comment into the /// comments viewable by the customer, effectively removing the entire text, which seems quite a bad customer experience), but please ensure you're okay with that.